core/uploadDiveLogsDE.cpp

```diff
@@ -196,6 +196,15 @@ bool uploadDiveLogsDE::prepareDives(const QString &tempfile, bool selected)
 }
 
 
+void uploadDiveLogsDE::cleanupTempFile()
+{
+	if (tempFile.isOpen()) {
+		tempFile.close();
+		tempFile.remove();
```

Comment:
so if the file is not open, we don't remove it? I'm confused.

Reply:
also, `.remove()` closes the file if necessary, according to the Qt documentation, so the `.close()` call is redundant.